contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -2063,7 +2103,7 @@ bool USDScene::hasChild( const SceneInterface::Name &name ) const
 
 void USDScene::childNames( SceneInterface::NameList &childNames ) const
 {
-	for( const auto &i : m_location->prim.GetAllChildren() )
+	for( const auto &i : m_location->prim.GetFilteredChildren( pxr::UsdTraverseInstanceProxies() ) )
```

Comment:
This seems unrelated to the topic of this commit. What does it do?

Reply:
It shouldn't have been in the PR and it's for my next PR to expand instanced USD geometry.